packages/edit-site/src/components/sidebar-dataviews/default-views.js

```diff
@@ -7,10 +7,27 @@ import { trash } from '@wordpress/icons';
 /**
  * Internal dependencies
  */
-import { LAYOUT_TABLE, OPERATOR_IN } from '../../utils/constants';
+import {
+	LAYOUT_LIST,
+	LAYOUT_TABLE,
+	LAYOUT_GRID,
+	OPERATOR_IN,
+} from '../../utils/constants';
+
+export const DEFAULT_CONFIG_PER_VIEW_TYPE = {
```

Comment:
This was moved from `page-pages/index.js` to initialize the `DEFAULT_VIEWS` with the layout data, which was not being initialized.